localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -1,29 +1,331 @@
+ count_feudal_female_wildhammer:0 "Chieftess"
+ county_feudal_wildhammer:0 "Chiefdom"
+ duke_feudal_male_wildhammer:0 "Thane"
+ duke_feudal_female_wildhammer:0 "Thane"
+ duchy_feudal_wildhammer:0 "High Chiefdom"
+ king_feudal_male_wildhammer:0 "High Thane"
+ king_feudal_female_wildhammer:0 "High Thane"
+
+ ### Dark Iron ###
+ # Feudal
+ count_feudal_male_darkiron:0 "Lord"
+ count_feudal_female_darkiron:1 "Lady"
+ county_feudal_darkiron:0 "Lordship"
+ duke_feudal_male_darkiron:0 "Thane"
+ duke_feudal_female_darkiron:0 "Thane"
+
+ ### Old Dwarven / Frostborn ###
+ # Feudal
+ count_feudal_male_old_dwarven:0 "Chief"
+ count_feudal_female_old_dwarven:0 "Chieftess"
+ county_feudal_old_dwarven:0 "Chiefdom"
+ duke_feudal_male_old_dwarven:0 "Thane"
+ duke_feudal_female_old_dwarven:0 "Thane"
+ duchy_feudal_old_dwarven:0 "High Chiefdom"
+ king_feudal_male_old_dwarven:0 "High Thane"
+ king_feudal_female_old_dwarven:0 "High Thane"
+ emperor_feudal_male_old_dwarven:0 "High King"
+ emperor_feudal_female_old_dwarven:0 "High Queen"
+ empire_feudal_old_dwarven:0 "High Kingdom"
+
+ ### Vrykul and Kvaldir ###
+ # Feudal
+ count_feudal_male_vrykul:0 "Thane"
+ count_feudal_female_vrykul:0 "Thane"
+ county_feudal_vrykul:0 "Chiefdom"
+ duke_feudal_male_vrykul:0 "Jarl"
+ duke_feudal_female_vrykul:0 "Frue"
+ duchy_feudal_vrykul:0 "Jarldom"
+ emperor_feudal_male_vrykul:0 "High King"
+ emperor_feudal_female_vrykul:0 "High King"
+ empire_feudal_vrykul:0 "High Kingdom"
+
+ ### Frost Vrykul ###*
+ # Feudal
+ count_feudal_male_frost_vrykul:0 "Thane"
+ count_feudal_female_frost_vrykul:0 "Thane"
+ county_feudal_frost_vrykul:0 "Chiefdom"
+ duke_feudal_male_frost_vrykul:0 "Overthane"
+ duke_feudal_female_frost_vrykul:0 "Overthane"
+ duchy_feudal_frost_vrykul:0 "Jarldom"
+ emperor_feudal_male_frost_vrykul:0 "High King"
+ emperor_feudal_female_frost_vrykul:0 "High King"
+ empire_feudal_frost_vrykul:0 "High Kingdom"
+
+ ### Staroveric ###
+ # Feudal
+ duke_feudal_male_solstice:0 "Grand Prince"
+ duke_feudal_female_solstice:0 "Grand Princess"
+ duchy_feudal_solstice:0 "Grand Principality"
+ emperor_feudal_male_solstice:0 "Tsar"
+ emperor_feudal_female_solstice:0 "Tsaritsa"
+ empire_feudal_solstice:0 "Tsardom"
+
+ ### Dryad group ###
+ # Feudal / Republic / Clan / Theocratic / Tribal
+ baron_male_dryad_group:0 "Tender"
+ baron_female_dryad_group:0 "Tender"
+ barony_dryad_group:0 "Refuge"
+ count_male_dryad_group:0 "Guardian"
+ count_female_dryad_group:0 "Guardian"
+ county_dryad_group:0 "Grove"
+ duke_male_dryad_group:0 "Lord"
+ duke_female_dryad_group:0 "Lady"
+ duchy_dryad_group:0 "Grand Grove"
+ king_male_dryad_group:0 "High Lord"
+ king_female_dryad_group:0 "High Lady"
+ kingdom_dryad_group:0 "Domain"
+ emperor_male_dryad_group:0 "King"
+ emperor_female_dryad_group:0 "Queen"
+ empire_dryad_group:0 "Kingdom"
+
+ ### Tol'vir ###
+ # Feudal - Normal
+ count_feudal_male_tolvir:0 "Governor"
+ count_feudal_female_tolvir:0 "Governor"
+ county_feudal_tolvir:0 "Region"
```

Comment:
I'm against using the titles governor, prefect. It fits republics, not feudals.
P.S. This applies not only to the tol'vir, but also to other races.

Reply:
Again it's inspired by some real titles. I have trouble seeing the tol'virs called "counts" while the dukes are called "nomes"